test-projects/social-login/server.js

```diff
@@ -0,0 +1,89 @@
+const keystone = require('@keystone-alpha/core');
+const { endAuthedSession } = require('@keystone-alpha/session');
+const {
+  facebookAuthEnabled,
+  githubAuthEnabled,
+  twitterAuthEnabled,
+  googleAuthEnabled,
+  wpAuthEnabled,
+  port,
+  staticRoute,
+  staticPath,
+} = require('./config');
+const { configureFacebookAuth } = require('./facebook');
+const { configureGitHubAuth } = require('./github');
+const { configureTwitterAuth } = require('./twitter');
+const { configureGoogleAuth } = require('./google');
+const { configureWPAuth } = require('./wordpress');
+
+const initialData = require('./data');
+
+keystone
+  .prepare({ port })
+  .then(async ({ server, keystone: keystoneApp }) => {
+    if (facebookAuthEnabled) {
+      configureFacebookAuth(keystoneApp, server);
```

Comment:
These `configure*` functions will no longer need a `server` parameter.

Reply:
still needs it to pass down to `setupAuthRoutes`

Reply:
removed it now. removed `setupAuthRoutes` form `configure*()`